sormas-rest/src/main/java/de/symeda/sormas/rest/CaseResource.java

```diff
@@ -45,6 +45,12 @@
 		return FacadeProvider.getCaseFacade().getAllActiveCasesAfter(new Date(since));
 	}
 
+	@GET
+	@Path("/allIncludeAdditionalParams/{since}")
+	public List<CaseDataDto> getAllCasesInCludeAdditionalObjects(@PathParam("since") long since) {
```

Comment:
Rename to getAllCasesWithExtendedChangeDateFilters

Reply:
changed